docs/rules/comma-dangle.md

```diff
@@ -12,6 +12,26 @@ var foo = {
 ```
 
 On the other hand, trailing commas simplify adding and removing items to objects and arrays, since only the lines you are modifying must be touched.
+Additionally, another argument in favor of trailing commas is that it improves the clarity of diffs when an item is added or removed from an object or array:
```

Comment:
Both points are needed, for sure, so agree on 2 sentences.

Do you think the meaning remains clear without “On the other hand” and “Additionally”?